send2cgeo.user.js

```diff
@@ -336,9 +323,10 @@ s.textContent = '(' + function() {
 
         var caches = $(".cache-details");
         caches.each(addSend2cgeoColumn);
+    }
 
-    } else if (document.getElementById('ctl00_ContentBody_CoordInfoLinkControl1_uxCoordInfoCode') !== null) {
-        // geocaching.com cache detail page
+    // Send to c:geo on cache detail page
+    if (document.location.href.match(/\.com\/(seek\/cache_details\.aspx|geocache\/)/) && !document.location.href.match(/^\/(geocache\/).*\/log/)) {
```

Comment:
which adress will be excluded with
~~~
!document.location.href.match(/^\/(geocache\/).*\/log/)
~~~
Maybe it can excluded by the @exclude directive.

Reply:
It's exclude the new loging page. But I just saw that it is not necessary at all.